CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 
 ## v2.3.9 - TBD
```

Comment:
I think this change warrants bumping the minor version, but I have an open PR that will already do this.